src/fides/api/api/deps.py

```diff
@@ -70,27 +66,3 @@ def get_cache() -> Generator:
             "Application redis cache required, but it is currently disabled! Please update your application configuration to enable integration with a Redis cache."
         )
     yield get_redis_connection()
-
-
-def get_messaging_service(
```

Comment:
just noting for self/posterity that these were not removed, but rather moved over to `api/service/deps.py` 👍